source/linux/Foundational_Components/U-Boot/UG-Memory.rst

```diff
@@ -1,973 +1,1120 @@
 
-    Note: On boards like AM57x GP EVM or BeagleBoard x15, where the second
-    USB instance is used as USB client, replace "dfu 0 mmc X" with "dfu 1
-    mmc X"
+   Note: On boards like AM57x GP EVM or BeagleBoard x15, where the second
+   USB instance is used as USB client, replace "dfu 0 mmc X" with "dfu 1
+   mmc X"
 
-    On the host side: binaries in FAT:
+   On the host side: binaries in FAT:
 
-    .. code-block:: console
+   .. code-block:: console
 
-        $ sudo dfu-util -D MLO -a MLO
-        $ sudo dfu-util -D u-boot.img -a u-boot.img
-        $ sudo dfu-util -D dra7-evm.dtb -a spl-os-args
-        $ sudo dfu-util -D uImage -a spl-os-image
+      $ sudo dfu-util -D MLO -a MLO
+      $ sudo dfu-util -D u-boot.img -a u-boot.img
+      $ sudo dfu-util -D dra7-evm.dtb -a spl-os-args
+      $ sudo dfu-util -D uImage -a spl-os-image
 
-    raw binaries:
+   raw binaries:
 
-    .. code-block:: console
+   .. code-block:: console
 
-        $ sudo dfu-util -D MLO -a MLO.raw
-        $ sudo dfu-util -D u-boot.img -a u-boot.img.raw
-        $ sudo dfu-util -D dra7-evm.dtb -a spl-os-args.raw
-        $ sudo dfu-util -D uImage -a spl-os-image.raw
+      $ sudo dfu-util -D MLO -a MLO.raw
+      $ sudo dfu-util -D u-boot.img -a u-boot.img.raw
+      $ sudo dfu-util -D dra7-evm.dtb -a spl-os-args.raw
+      $ sudo dfu-util -D uImage -a spl-os-image.raw
 
-    If the binaries are files in a fat partition, you need to specify their
-    name if they differ from the default values ("uImage" and "args"). Note
-    that DFU uses the names "spl-os-image" and "spl-os-args", so this step
-    is required in the case of DFU. From u-boot prompt
+   If the binaries are files in a fat partition, you need to specify their
+   name if they differ from the default values ("uImage" and "args"). Note
+   that DFU uses the names "spl-os-image" and "spl-os-args", so this step
+   is required in the case of DFU. From u-boot prompt
 
-    .. code-block:: console
+   .. code-block:: console
 
-        => setenv falcon_image_file spl-os-image
-        => setenv falcon_args_file spl-os-args
-        => saveenv
+      => setenv falcon_image_file spl-os-image
+      => setenv falcon_args_file spl-os-args
+      => saveenv
 
-    Set the environment variable "boot\_os" to 1. From u-boot prompt
+   Set the environment variable "boot\_os" to 1. From u-boot prompt
 
-    .. code-block:: console
+   .. code-block:: console
 
-        => setenv boot_os 1
-        => saveenv
+      => setenv boot_os 1
+      => saveenv
 
-    Set the board boot from SD (or eMMC respectively) and reset the EVM. The
-    SPL directly boots the kernel image from SD (or eMMC).
+   Set the board boot from SD (or eMMC respectively) and reset the EVM. The
+   SPL directly boots the kernel image from SD (or eMMC).
 
-Steps for working around SD card issues
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Steps for working around SD card issues in u-boot
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
This section is almost exactly the same as the above section for linux. The only addition is `Increase power cycle period`. May be worth referencing it and just mentioning the new step.

Reply:
To be honest, I would rather keep the section as it is. The section is not that large and I think there are already to many re-directions on this page

Reply:
As a reader and reviewer, reiterating the same content over again in something this long is both unnecessary and frustrating.

Reply:
I think there is a confusion here. The "steps for working around SD card issues in u-boot" is a section specifically for the UG-Memory.rst file which targets u-boot. So we use examples in u-boot.

The "steps for working around SD card issues in Linux" is a section for the MMC-SD.rst which targets Linux, so we use examples with Linux